src/Data/Ticket/TicketEditingConfig.ts

```diff
@@ -8,11 +8,15 @@ provideEditingConfig(Ticket, {
     description: { title: 'Description' },
     number: { title: 'Ticket number' },
     referenceNumber: { title: 'Reference number' },
-    type: { title: 'Type' },
-    status: { title: 'Status' },
+    type: { title: 'Type (enum)' },
```

Comment:
* [ ] FYI the term "enum" is developer specific and editors may fail to understand it. To a certain extent this also applies to "localized". Ideas:
  * Type (code), Type (human readable)
  * Type code, Type description